attributes/default.rb

```diff
@@ -143,6 +143,9 @@
 
 # Configure Abiquo websockify
 default['abiquo']['websockify']['port'] = 41337
+default['abiquo']['websockify']['api_url'] = 'https://localhost/api'
```

Comment:
Actually, since we now use the config properties, would it make sense such resolving? Since we now can configure arbitraty config items in UI and UI and WS profiles can be split across hosts we could just set the property in UI in the nodes that include the UI recipe and the WS one on the ones using only the WS recipe. Of course, on a server node you will need to set it twice. WDYT?